File: ClassificacaoNumero.java
//- um algoritmo que leia uma lista de números e classifique-os em ordem crescente.

import java.util.ArrayList;
import java.util.Collections;
import java.util.Scanner;

public class ClassificacaoNumero {
public static void main(String args[]){
Scanner sc = new Scanner(System.in);
ArrayList <Integer> numeros = new ArrayList<>();



System.out.println("Escreva um numero por vez. Para encerrar digite qualquer letra: ");
 while (sc.hasNextInt()) {
 int numero = sc.nextInt();
 numeros.add(numero);
 }

 Collections.sort(numeros);

System.out.println(numeros);

    sc.close();
}


}
    

File: Classificacao_numero.py
# um algoritmo que leia uma lista de números e classifique-os em ordem crescente.
lista = []

while True:
    entrada = input("Digite um numero ou stop para encerrar: \n")
    if entrada.lower() == "stop":
        break
    try:
        numero = int (entrada)
    
    except ValueError:
       print("Por favor, digite um número inteiro válido.\n")
       continue
      
    lista.append(numero)
    
print("\nOs numeros são: ")
print(lista)    
